roles/rabbitmq/tasks/main.yml

```diff
@@ -44,5 +44,8 @@
 - name: correct rabbitmqadmin modes
   file: group=root owner=root mode=0755 path=/usr/local/bin/rabbitmqadmin
 
+- name: permit access to rabbitMQ server
+  ufw: rule=allow to_port={{ rabbitmq.port }} proto=tcp
```

Comment:
Updated code to allow port from required hosts(computes) in the commit 
https://github.com/persistent-ursula/ursula/commit/86542b81388d7b765e2ebe13f55b654f2efe3871